### PR TITLE
feat: creating script to send a test email of a mailchimp campaign

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

File: mailchimp/test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+    campaignId = campaign.id;
+  } else {
+    // Value is the campaign id
+    campaignId = value
+  }
+
+  // Send test email
+  try {
+    const response = await sendTestEmail({
```
Comment (obiyankenobi) [inline mailchimp/test.js:53]: `sendTestEmail ` doesn't need to have `async`?

File: mailchimp/README.md [hunk 2]
```diff
@@ -8,4 +8,11 @@
 2. Generate html file for the 'This Week in Hathor' week post;
 3. Execute `node index.js number htmlPath`, where `number` is the week number and `htmlPath` is the path to the html file.
 
+## Send test email
+
+You need the email to send and the campaign id or name.
```
Comment (obiyankenobi) [inline mailchimp/README.md:13]: I was a bit confused at first what you meant by 'you need the email'. I thought it was the email code (html/css). Also, `to_email` was not very clear. What about?
```
You need the destination email address and the campaign id or name.

`node test.js email_address id campaignId` or `node test.js email_address name campaignName`
```

File: mailchimp/test.js (hunk 1 above)
Comment (pedroferreira1) [reply mailchimp/test.js:53]: No need to. The method needs `async` only if it has an `await` call inside it. The `sendTestEmail` method returns a promise.

File: mailchimp/README.md (hunk 2 above)
Comment (pedroferreira1) [reply mailchimp/README.md]: Yeah, it's better. Changed it

File: mailchimp/test.js (hunk 1 above)
Comment (jansegre) [reply mailchimp/test.js:53]: I'm OK leaving out the `async`, but I'm also OK adding it (and an await) which would make it more obvious that it returns a promise. But I'm really fine either way.

Comment (pedroferreira1) [reply mailchimp/test.js]: I will leave like this for now. To make more obvious, we could add the docstring

**Comment by pedroferreira1** [comment]
> I'd probably prefer to rename test.js to something else, like dry_run.js, beta.js, simulate.js or anything else that is obviously not a unittest.

Renamed the file to `simulate.js`

**Pull request merged**